src/appMain/hmi_capabilities.json

```diff
@@ -467,9 +471,234 @@
                         "rdsDataAvailable": true,
                         "signalChangeThresholdAvailable": true,
                         "signalStrengthAvailable": true,
+                        "sisDataAvailable":true,
                         "stateAvailable": true
                     }
-                ]
+                ],
+                "audioControlCapabilities": [
+                {
+                        "moduleName": "audio",
+                        "sourceAvailable": true,
+                        "volumeAvailable": true,
+                        "equalizerAvailable": true,
+                        "keepContextAvailable" : true,
+                        "equalizerMaxChannelId": 80
```

Comment:
suggest to make it a small number, like 10

Reply:
@yang1070 Done b1bcb4f